platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3563,13 +3563,12 @@
             });
         };
 
-        $scope.ngFormNameAttrSuffix = "form";
```

Comment:
From what I recall this suffix was added because of issues with the name duplicating based on the name of the field. Should this get removed?